### DISCUSSION TITLE
Programmatically expand row from another location

### DISCUSSION THREAD

**Discussion by SonicMagna** [binodswain/react-faq-component#44] [Q&A]
Is there a way to programmatically expand a FAQ row?
Usage would be having a link elsewhere in the App pointing to named anchor, a specific FAQ row, but also being able to pass something that would expand that row (Calling the expand function that is used in the onClick).
I looked through the module's code, but it's minified and I was not able to discern how to accomplish this.
I also experimented with the tabFocus setting, but did not succeed in getting any results.
Thanks

**Comment by binodswain** [comment]
The "programmatically expand a FAQ row" feature was not available previously.
I have added it in the latest version, and have tested limited use cases.

View here (last example): https://binodswain.github.io/react-faq-component/demo-more

Another example I have added in readme. https://github.com/binodswain/react-faq-component#getrowoptions-props-optional
